testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1563,3 +1563,29 @@
   raise 'File injection failed' unless return_code.zero?
   node.run('salt-call --local --file-root=/usr/share/susemanager/salt --module-dirs=/usr/share/susemanager/salt/ --log-level=info --retcode-passthrough state.apply ' + state)
 end
+
+And(/^I upload "([^\"]*)" to "([^\"]*)" on "([^\"]*)"$/) do |file, targetdir, host|
+  target = get_target(host)
+  source = File.dirname(__FILE__) + '/../upload_files/' + file
+  dest = targetdir + File.basename(file)
+  return_code = file_inject(target, source, dest)
+  raise 'File injection failed' unless return_code.zero?
+  target.run("chmod 644 #{dest}")
+end
+
+And(/^I upload autoinstall mocked files on ([^\"]*)$/) do |host|
```

Comment:
`When`

Reply:
We have a code style (it might be not documented, our bad) about passing the string parameters in our steps, we use to enclose them between double-quotes.
```suggestion
And(/^I upload autoinstall mocked files on "([^\"]*)"$/) do |host|
```
In any case, as we will not upload those mocked files anywhere else, I propose that we avoid it, and just have something like:
```suggestion
And(/^I upload autoinstall mocked files on the server$/) do
```
Then as target, you can directly use the global variable $server, example: `$server.run(...)`